opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/Ctags.java

```diff
@@ -567,7 +567,7 @@ private void readTags(CtagsReader reader) throws InterruptedException {
         } catch (Exception e) {
             LOGGER.log(Level.WARNING, "CTags parsing problem: ", e);
         }
-        LOGGER.severe("CTag reader cycle was interrupted!");
+        LOGGER.finest("CTag reader cycle was interrupted!");
```

Comment:
Could "!" be a keyword?

Reply:
yes.

Reply:
Here is a log invocation having level `FINE` and three "!". "!" may not be a keyword.
https://github.com/hms-dbmi/IRCT-API/blob/aa0f039f454834e78affc94e159bb76a1ffefed1/src/main/java/edu/harvard/hms/dbmi/bd2k/irct/controller/ResultController.java#L250

Reply:
Maybe "interrupt" could be a keyword.